### ISSUE TITLE
Starring a message creates task

### ISSUE THREAD

**Issue by hluetck** [Ericsson/exchangecalendar#218]
I just updated to the newest beta release and find that all my starred mails are now added as tasks. Starring a new mail also creates a task. This did not happen in the previous version. Is it possible to disable this somewhere?

Thanks, Henry

**Comment by bavincen** [comment]
its email followup :)

**Comment by bavincen** [comment]
We didn’t add option to disable this feature now.

**Comment by hluetck** [comment]
I would say that starring a message marks it as important, which does not necessarily imply a follow-up. I definitely think that it is useful to create a task from an email (follow-up), but in my opinion this should be separate from highlighting (starring) a message.

**Comment by jesheldon** [comment]
I would also like to see this go back to the old behavior of showing only tasks, or else having some setting to filter out those starred emails from the tasks list. I realize this is default behavior at the exchange server (i.e. Office365 adds starred emails to task list), but for some users (like me) this behavior makes the task list almost useless. I updated to 3.4.0~beta7 to get the fix for calendar events updating only after TBird restart, but now the Task list is not behaving as desired.

**Comment by bavincen** [comment]
Please clear cache.

**Comment by jesheldon** [comment]
I tried that. All the items including starred messages just get reloaded 
as tasks after the cache is cleared. Let me know if there is any logging 
I could do that would help troubleshoot.

Joan

**Comment by bavincen** [comment]
We have added a option to disable follow-up task

**Comment by bavincen** [comment]
https://github.com/Ericsson/exchangecalendar/wiki/Followup-Task-Settings

**Comment by jesheldon** [comment]
Thanks, but I can't figure out how to find and drop in any updated files 
to be able to get this EWS Settings (Mail) tab. Will there be a new 
version released soon with this new feature?

Joan
